c2corg_ui/templates/sidemenu.html

```diff
@@ -49,7 +49,7 @@
+      <iframe src="${request.static_url('c2corg_ui:static/affiz.html')}" width="160" height="320" frameborder="0" scrolling="no"></iframe>
```

Comment:
Why do you use an iframe? Don't you directly want to insert the ads script into the main page? This would avoid an additional request.